barretenberg/cpp/src/barretenberg/stdlib/primitives/safe_uint/safe_uint.hpp

```diff
@@ -29,11 +29,10 @@ template <typename Builder> class safe_uint_t {
         : value(value)
         , current_max(current_max)
     {
-        ASSERT(safety == IS_UNSAFE);
-        if (current_max > MAX_VALUE) // For optimal efficiency this should only be checked while testing a circuit
```

Comment:
Not sure what it means by "testing a circuit". Looks like it's already checked in release builds anyway?